_data/changelogs/component-date-range-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date range picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added `aria-disabled` to the list of expected attributes. 
+    summaryAdditional: Now, the component will disable toggle when the `aria-disabled` attribute is present.
+    affectsAccessibility: true
+    affectsJavascript: true
+    githubPr: 2817
```

Comment:
**todo:** Looks like this was set to the changelog PR number instead of the USWDS contribution!

```suggestion
    githubPr: 6013
```

Reply:
Resolved in [b87011c](https://github.com/uswds/uswds-site/pull/2817/commits/b87011c032d02d404e2f4298ffeac90993efc4d3)